homeassistant/components/automation/__init__.py

```diff
@@ -94,15 +92,13 @@ def _platform_validator(config):
 _CONDITION_SCHEMA = vol.All(cv.ensure_list, [cv.CONDITION_SCHEMA])
 
 PLATFORM_SCHEMA = vol.All(
-    cv.deprecated(CONF_HIDE_ENTITY, invalidation_version="0.107"),
     vol.Schema(
         {
             # str on purpose
             CONF_ID: str,
             CONF_ALIAS: cv.string,
             vol.Optional(CONF_DESCRIPTION): cv.string,
             vol.Optional(CONF_INITIAL_STATE): cv.boolean,
-            vol.Optional(CONF_HIDE_ENTITY, default=DEFAULT_HIDE_ENTITY): cv.boolean,
```

Comment:
Thinking here, is it better that automations don't load after an update or that automation entities no longer hide? As in, what if we remove the functionality of `hide_entity` but we keep the `cv.deprecated` and `vol.Optional` around for a couple of months. There is no cost to us and it is more user friendly.

Reply:
Added the option back in, set the deprecation release to 0.110, which should give 2+ months.